thirdparty/faiss/faiss/IndexFlatElkan.cpp

```diff
@@ -15,12 +15,15 @@
 
 #include <faiss/impl/FaissAssert.h>
 #include <faiss/utils/distances.h>
+#include <faiss/utils/utils.h>
 
 namespace faiss {
 
-IndexFlatElkan::IndexFlatElkan(idx_t d, MetricType metric, bool is_cosine, bool use_elkan)
+IndexFlatElkan::IndexFlatElkan(idx_t d, MetricType metric, bool is_cosine, bool use_elkan, float* data)
```

Comment:
?! `data` which is passed as a parameters is not used anywhere

Reply:
Yeah, it's not required to be set by callers. Remove it from the interface.